requirements/dev.txt

```diff
@@ -1,5 +1,9 @@
 # Dependencies needed only for development/testing.
 
+codecov==2.0.15 \
```

Comment:
It seems that `codecov` depends on the package `coverage`.
That means you will need to also include that package (since it is part of the hierarchy) and its associated hash.